bin/stable/prepare_project_edition.sh

```diff
@@ -57,7 +57,7 @@ sudo sed -i "s/\['test' => true\]/\['test' => true, 'behat' => true\]/g" config/
 cp "behat_ibexa_${PROJECT_EDITION}.yaml" behat.yaml
 
 if [[ $PHP_IMAGE == *"8.2"* ]] || [[ $PHP_IMAGE == *"8.3"* ]]; then
```

Comment:
I think this part can be removed - the error handler will be included by default in the skeletons (and if it wasn't it would be a problem)

Reply:
Done in https://github.com/ibexa/ci-scripts/pull/91/commits/f5420d41434ed1aa16b484d44f88cb7bb23ff8c4. Thank you.